PR: https://github.com/odoo/odoo/pull/

From: 7978d4ba0f093c4b5400dbb1776ff5c0416e203b
From: Yannick Tivisse
Date: 2018-09-10 12:23:42

Breaking data model changes score: 4, change matches:
-    _inherit = 'res.users'
-    has_group_hr_attendance = fields.Boolean(
-    has_group_hr_attendance_use_pin = fields.Boolean(
-    group_hr_attendance_user = fields.Selection(

Total Changes: 24

Revert "[IMP] *: Define groups on res.users models"

This reverts commit 99f497b390e680629dce3a7e2e282ada74ee9884.

================================= pseudo patch: =================================

--- a/addons/hr_attendance/models/__init__.py
+++ b/addons/hr_attendance/models/__init__.py
@@ -3,4 +3,3 @@
 from . import res_config_settings
 from . import hr_attendance
 from . import hr_employee
-from . import res_users

--- a/addons/hr_attendance/models/res_users.py
+++ b/None
@@ -1,23 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import fields, models
-
-
-class ResUsers(models.Model):
-    _inherit = 'res.users'
-
-    has_group_hr_attendance = fields.Boolean(
-        'Manual Attendance', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='hr_attendance.group_hr_attendance',
-        help='The user will gain access to the human resources attendance menu, enabling him to manage his own attendance.')
-
-    has_group_hr_attendance_use_pin = fields.Boolean(
-        'Enable PIN use', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='hr_attendance.group_hr_attendance_use_pin',
-        help='The user will have to enter his PIN to check in and out manually at the company screen.')
-
-    group_hr_attendance_user = fields.Selection(
-        selection=lambda self: self._get_group_selection('base.module_category_hr_attendance'),
-        string='Attendance', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        category_xml_id='base.module_category_hr_attendance')
